### ISSUE TITLE
Regression spotted with 11.0.13+8 and 11.0.12+7 on macOS (Catalina)

### ISSUE THREAD

**Issue by JDESTM** [adoptium/adoptium-support#425]
## Summary

Using Adoptium JRE 11.0.13+8 and 11.0.12+7, **on macOS Catalina**, our application hangs in native code when invoking CGraphicsDevice.nativeGetScreenInsets(int), whereas same code works just fine with AdoptOpenJDK 11.0.11+9.

Note: issue reproduced also with Zulu JRE.
Note: not reproduced on BigSur (I don't have access to Monterrey to perform same test).

## Steps to reproduce

1. build attached snippet (eclipse rcp) with Maven: [com.st.demoapps.zip](https://github.com/adoptium/adoptium-support/files/7821349/com.st.demoapps.zip)
```
cd  com.st.demoapps
mvn clean verify
```
2. execute it using above combination of JRE versions
```
./releng/com.st.demoapps.product/target/products/com.st.demoapps.product/macosx/cocoa/x86_64/Eclipse.app/Contents/MacOS/eclipse
```

JRE can be configured within eclipse.ini file:
```
./releng/com.st.demoapps.product/target/products/com.st.demoapps.product/macosx/cocoa/x86_64/Eclipse.app/Contents/Eclipse/eclipse.ini
```

add the following lines prior to -vmargs line:
```
-vm
<JAVA_PATH>/Contents/Home/lib/jli/libjli.dylib
```

3. notice that the UI is never displayed if using 11.0.13+8 (or 11.0.12+7) on Catalina and that the java stack looks like:
```
Thread [AWT-EventQueue-0] (Suspended)
        CGraphicsDevice.nativeGetScreenInsets(int) line: not available [native method]
        CGraphicsDevice.getScreenInsets() line: 133
        LWCToolkit.getScreenInsets(GraphicsConfiguration) line: 487
        CViewEmbeddedFrame(Window).init(GraphicsConfiguration) line: 518
        CViewEmbeddedFrame(Window).<init>() line: 549
        CViewEmbeddedFrame(Frame).<init>(String) line: 423
        CViewEmbeddedFrame(Frame).<init>() line: 388
        CViewEmbeddedFrame(EmbeddedFrame).<init>(long, boolean) line: 95
        CViewEmbeddedFrame(EmbeddedFrame).<init>(long) line: 92
        CViewEmbeddedFrame(EmbeddedFrame).<init>() line: 80
        CViewEmbeddedFrame.<init>(long) line: 48
        SWT_AWT.initFrame(long, String) line: not available [native method]
        SWT_AWT$1.run() line: 154
        InvocationEvent.dispatch() line: 313
        EventQueue.dispatchEventImpl(AWTEvent, Object) line: 770
        EventQueue$4.run() line: 721
        EventQueue$4.run() line: 715
        AccessController.doPrivileged(PrivilegedAction<T>, AccessControlContext) line: not available [native method]
        ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(PrivilegedAction<T>, AccessControlContext, AccessControlContext) line: 85
        EventQueue.dispatchEvent(AWTEvent) line: 740
        EventDispatchThread.pumpOneEventForFilters(int) line: 203
        EventDispatchThread.pumpEventsForFilter(int, Conditional, EventFilter) line: 124
        EventDispatchThread.pumpEventsForHierarchy(int, Conditional, Component) line: 113
        EventDispatchThread.pumpEvents(int, Conditional) line: 109
        EventDispatchThread.pumpEvents(Conditional) line: 101
        EventDispatchThread.run() line: 90
```

If you have access to different macOS version notice also that the same use-case works fine on BigSur (and Monterrey?)

## Expected results

Application should not hangs are startup.

## Actual results

No UI visible, only an icon in macOS dock.

## Triaging info

_Java version:_
```
openjdk version "11.0.13" 2021-10-19
OpenJDK Runtime Environment Temurin-11.0.13+8 (build 11.0.13+8)
OpenJDK 64-Bit Server VM Temurin-11.0.13+8 (build 11.0.13+8, mixed mode)
```

_What is your operating system and platform?_
macOS Catalina 10.15.3 on x86-64

_How did you install Java?_
ZIP archive picked on https://adoptium.net/

_Did it work before?_
Changing the runtime environment is enough to trigger the issue.

_Did you test with other Java versions?_
```
openjdk version "11.0.13" 2021-10-19 LTS
OpenJDK Runtime Environment Zulu11.52+51-CA (build 11.0.13+8-LTS)
OpenJDK 64-Bit Server VM Zulu11.52+51-CA (build 11.0.13+8-LTS, mixed mode)

openjdk version "11.0.12" 2021-07-20
OpenJDK Runtime Environment Temurin-11.0.12+7 (build 11.0.12+7)
OpenJDK 64-Bit Server VM Temurin-11.0.12+7 (build 11.0.12+7, mixed mode)
```
➡️ Fails
```
openjdk version "11.0.11" 2021-04-20 LTS
OpenJDK Runtime Environment Zulu11.48+21-CA (build 11.0.11+9-LTS)
OpenJDK 64-Bit Server VM Zulu11.48+21-CA (build 11.0.11+9-LTS, mixed mode)

openjdk version "11.0.11" 2021-04-20
OpenJDK Runtime Environment AdoptOpenJDK-11.0.11+9 (build 11.0.11+9)
OpenJDK 64-Bit Server VM AdoptOpenJDK-11.0.11+9 (build 11.0.11+9, mixed mode)
```
➡️ Works
```
openjdk version "17.0.1" 2021-10-19
OpenJDK Runtime Environment Temurin-17.0.1+12 (build 17.0.1+12)
OpenJDK 64-Bit Server VM Temurin-17.0.1+12 (build 17.0.1+12, mixed mode, sharing)
```
➡️ Works but we're currently stuck on Java 11 LTS.

**Comment by karianna** [comment]
@JDESTM Are you able to quickly try one of the nightlies at https://adoptium.net/nightly.html?variant=openjdk11&jvmVariant=hotspot

**Comment by JDESTM** [comment]
Thanks @karianna ! Yes I confirm this nightly build solves the hang/freeze at startup 👍 

By curiosity do you have any reference to the initial issue triggering a fix in the version to come ?

**Comment by karianna** [comment]
I'm not 100% sure but there are ~20 AWT/Mac OS X fixes in 11.0.14 / 17.0.2. The full release notes will come out in late Jan
